File: app/src/main/java/com/example/bhagavadgita/data/model/Chapters.java
package com.example.bhagavadgita.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Chapters {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("slug")
    @Expose
    private String slug;
    @SerializedName("name_transliterated")
    @Expose
    private String nameTransliterated;
    @SerializedName("name_translated")
    @Expose
    private String nameTranslated;
    @SerializedName("verses_count")
    @Expose
    private Integer versesCount;
    @SerializedName("chapter_number")
    @Expose
    private Integer chapterNumber;
    @SerializedName("name_meaning")
    @Expose
    private String nameMeaning;
    @SerializedName("chapter_summary")
    @Expose
    private String chapterSummary;
    @SerializedName("chapter_summary_hindi")
    @Expose
    private String chapterSummaryHindi;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public String getNameTransliterated() {
        return nameTransliterated;
    }

    public void setNameTransliterated(String nameTransliterated) {
        this.nameTransliterated = nameTransliterated;
    }

    public String getNameTranslated() {
        return nameTranslated;
    }

    public void setNameTranslated(String nameTranslated) {
        this.nameTranslated = nameTranslated;
    }

    public Integer getVersesCount() {
        return versesCount;
    }

    public void setVersesCount(Integer versesCount) {
        this.versesCount = versesCount;
    }

    public Integer getChapterNumber() {
        return chapterNumber;
    }

    public void setChapterNumber(Integer chapterNumber) {
        this.chapterNumber = chapterNumber;
    }

    public String getNameMeaning() {
        return nameMeaning;
    }

    public void setNameMeaning(String nameMeaning) {
        this.nameMeaning = nameMeaning;
    }

    public String getChapterSummary() {
        return chapterSummary;
    }

    public void setChapterSummary(String chapterSummary) {
        this.chapterSummary = chapterSummary;
    }

    public String getChapterSummaryHindi() {
        return chapterSummaryHindi;
    }

    public void setChapterSummaryHindi(String chapterSummaryHindi) {
        this.chapterSummaryHindi = chapterSummaryHindi;
    }
}

File: app/src/main/java/com/example/bhagavadgita/data/model/Chapters.kt
package com.example.bhagavadgita.data.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Entity(tableName = "chapters")
data class Chapters(
    @PrimaryKey()
    @ColumnInfo(name = "id")
    @SerializedName("id")
    @Expose
    var id: Int? = 0,

    @ColumnInfo(name = "name")
    @SerializedName("name")
    @Expose
    var name: String? = "",

    @ColumnInfo(name = "slug")
    @SerializedName("slug")
    @Expose
    var slug: String? = "",

    @ColumnInfo(name = "name_transliterated")
    @SerializedName("name_transliterated")
    @Expose
    var nameTransliterated: String? = "",

    @ColumnInfo(name = "name_translated")
    @SerializedName("name_translated")
    @Expose
    var nameTranslated: String? = "",

    @ColumnInfo(name = "verses_count")
    @SerializedName("verses_count")
    @Expose
    var versesCount: Int? = -1,

    @ColumnInfo(name = "chapter_number")
    @SerializedName("chapter_number")
    @Expose
    var chapterNumber: Int? = -1,

    @ColumnInfo(name = "name_meaning")
    @SerializedName("name_meaning")
    @Expose
    var nameMeaning: String? = "null",

    @ColumnInfo(name = "chapter_summary")
    @SerializedName("chapter_summary")
    @Expose
    var chapterSummary: String? = null,

    @ColumnInfo(name = "chapter_summary_hindi")
    @SerializedName("chapter_summary_hindi")
    @Expose
    var chapterSummaryHindi: String? = ""
) {
    constructor() : this(
        id = 0,
        name = null,
        slug = null,
        nameTransliterated = null,
        nameTranslated = null,
        versesCount =null,
        chapterNumber =null,
        nameMeaning = null,
        chapterSummary = null,
        chapterSummaryHindi = null
    )
}